x-pack/plugins/data_enhanced/public/search/sessions_mgmt/lib/get_expiration_status.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+import moment from 'moment';
+import { SessionsMgmtConfigSchema } from '../';
+
+export const getExpirationStatus = (config: SessionsMgmtConfigSchema, expires: string | null) => {
+  const tNow = moment.utc().valueOf();
+  const tFuture = moment.utc(expires).valueOf();
+
+  // NOTE this could end up negative. If server time is off from the browser's clock
+  // and the session was early expired when the browser refreshed the listing
```

Comment:
@lizozom @elastic-jb What do you think about this? Should we fake that the status is "Expired" in the UI if calculating `expiration time - now` is negative?

Reply:
Does the monitoring task take care of this @lizozom? I can't remember offhand